### PR TITLE
Composite resolver

### LINKED ISSUE #101: Добавить документацию для критериев нормальности
Добавить документацию для критериев нормальности с примерами

### PR DESCRIPTION
**Pull request by alex98247**
## Summary

Add documentation pages and navigation for goodness-of-fit criteria, and update limit distribution resolution fallback behavior.

Solve the issue: #101, #102

## Quick changelog

- Added criterion documentation for exponentiality, uniformity, beta, gamma, Laplace, Student, Weibull, log-normal, and normality goodness-of-fit statistics.
- Updated `mkdocs.yml` navigation to include the new structured documentation paths.
- Restored `CompositeLimitDistributionResolver` with local storage lookup followed by Monte Carlo fallback.
- Added resolver and constants tests.
- Fixed several Weibull statistic edge cases.

## What's new?

This PR expands the project documentation by moving existing exponentiality pages under `docs/criteria/goodness-of-fit/` and adding detailed pages for multiple distribution families and criteria.

It also updates the limit distribution resolver flow so composite resolution first attempts local storage and falls back to Monte Carlo calculation when no stored distribution is available. New tests cover local-hit and fallback behavior, storage resolver query construction, and database URL environment defaults.

Additional Weibull statistic fixes adjust Anderson-Darling execution arguments, broaden Laplace transform type matching, and normalize Cabana-Quiroz input to a NumPy array.

### CODE REVIEW

File: docs/criteria/goodness-of-fit/beta/anderson_darling_beta_criterion.md
Comment (f1i3g3) [inline docs/criteria/goodness-of-fit/beta/anderson_darling_beta_criterion.md:1]: Образец для документации в experiment.

File: src/pysatl_criterion/hypothesis_testing/limit_distribution/base.py
Comment (f1i3g3) [inline src/pysatl_criterion/hypothesis_testing/limit_distribution/base.py:1]: О, логгер!

File: src/pysatl_criterion/statistics/goodness_of_fit/weibull.py [hunk 1]
```diff
@@ -1286,10 +1286,10 @@ def execute_statistic(self, rvs, m=100, a=-5, _type="LT3_WEIBULL"):
         """
         n = len(rvs)
 
-        if _type == "LT2_WEIBULL":
+        if _type == "LT2_WEIBULL" or _type.startswith("LT2_"):
             t_values = np.linspace(-m, -1, num=m) / m
 
-        elif _type == "LT3_WEIBULL":
+        elif _type == "LT3_WEIBULL" or _type.startswith("LT3_"):
```
Comment (f1i3g3) [inline src/pysatl_criterion/statistics/goodness_of_fit/weibull.py:1292]: Не излишне ли, если мы для всех добавляем?

File: src/pysatl_criterion/statistics/goodness_of_fit/weibull.py [hunk 2]
```diff
@@ -1433,6 +1433,7 @@ def execute_statistic(self, rvs):
         :param rvs: array of observed data samples.
         :return: CQ test statistic value.
         """
+        rvs = np.asarray(rvs)
```
Comment (f1i3g3) [inline src/pysatl_criterion/statistics/goodness_of_fit/weibull.py:1436]: !

File: tests/utils/test_constants.py
Comment (f1i3g3) [inline tests/utils/test_constants.py:1]: +

Comment (f1i3g3) [reply tests/utils/test_constants.py:1]: Надо похожее добавить в experiment.

**Review by f1i3g3 — APPROVED** [review]
Проблем не вижу.

**Review by quepum — APPROVED** [review]
